projects/olympics/BarChart.jsx

```diff
@@ -129,7 +127,15 @@ export const BarChart = ({ yTitle = "COUNTRY", data, values }) => {
                       .slice(0, groupIdx)
                       .map((prevs) => row[prevs.property])
                       .reduce((a, b) => a + b, 0);
-
+                    const xScalePreviousPercent = xScale(previousPercentage) / 6;
+                    const xAnimation =
+                      previousPercentage > 0
+                        ? [MARGIN.LEFT + initialX, MARGIN.LEFT + xScalePreviousPercent]
+                        : MARGIN.LEFT;
+                    const xScaleRowValProperty = xScale(row[val.property]);
+                    const widthMotion = xScaleRowValProperty / 6;
+                    const widthAnimation = [initialWidth, widthMotion];
+                    const yAnimated = LABELS[idx] / 10 + barIdx * BAR_HEIGHT + 17 / 2;
```

Comment:
Could we remove the `LABELS` need and move to generate that array of values (or the value for each row itself) from the data that we passed?

Reply:
Good catch there. Fixed.